src/core_plugins/kibana/public/management/sections/settings/advanced_row.js

```diff
@@ -33,7 +33,7 @@ uiModules.get('apps/management')
       };
 
       $scope.maybeCancel = function ($event, conf) {
-        if ($event.keyCode === keyCodes.ESC) {
+        if ($event.keyCode === keyCodes.ESCAPE) {
```

Comment:
We need to replace the custom `keyCodes` instance here with `import { keyCodes } from 'ui_framework/services';`